data/reusables/gated-features/dependency-review.md

```diff
@@ -1,3 +1,3 @@
-{% ifversion fpt or ghec %}Dependency review is available for all public repositories and for private repositories owned by organizations where {% data variables.product.prodname_GH_advanced_security %} is enabled.{% endif %}
+{% ifversion fpt or ghec %}Dependency review is available for all public repositories. As well as private repositories owned by organizations where {% data variables.product.prodname_GH_advanced_security %} is enabled.{% endif %}
```

Comment:
I think that the initial text is clear but I'm open to updates if this caused confusion with our customers. What about this?
(Will wait to hear from @jhutchings1 before applying this suggestion)
```suggestion
{% ifversion fpt or ghec %}Dependency review is available for all public repositories, as well as well as private repositories owned by organizations where {% data variables.product.prodname_GH_advanced_security %} is enabled.{% endif %}
```

Reply:
yeah it caused some confusion, to be fair I even read it wrong 😮‍💨 I think the purpose is making it a little more apparent. I read this as you needed a GHAS licence to run this on both public repos and private repos. I think it makes sense to decouple it, like you have ❤️

Reply:
I like @mchammer01 's suggestion above. The content is accurate in regards to feature availability, but if we can make it more clear, we should.

Reply:
thanks, @jhutchings1 👍 @mchammer01 are we good to commit your suggestion, please? I think this pleases everyone ❤️ thank you all.